scripts/tests/test-ci-all.sh

```diff
@@ -33,6 +33,11 @@ cargo nextest run --no-run ${CARGO_PROFILE:+--cargo-profile ${CARGO_PROFILE}} ${
 # If you really need to break this rule, ping dpc
 export FM_CARGO_DENY_COMPILATION=1
 
+function rust_unit_tests() {
```

Comment:
Whoa, have we not been running unit tests in CI?

Reply:
They are being run in the Code Coverage step. But now that I'd like to run them multiple times it makes sense to run them here in parallel with everything as well.

Reply:
After this commit, should we undo running coverage tests 5 times? Seems like this should have it covered.